src/lib.rs

```diff
@@ -47,36 +47,36 @@ lazy_static! {
 }
 
 struct SplitCaptures<'r, 't> {
```

Comment:
Maybe we should rename this to `SplitMatches`?

Reply:
Done in https://github.com/messense/jieba-rs/commit/9c3f320f3c8d8e90bbee7b5891f4ec69c5acc725